packages/editor/src/components/block-mover/index.js

```diff
@@ -18,7 +18,8 @@ import { withInstanceId, compose } from '@wordpress/compose';
  * Internal dependencies
  */
 import { getBlockMoverDescription } from './mover-description';
-import { upArrow, downArrow } from './arrows';
+import { upArrow, downArrow, dragHandle } from './arrows';
```

Comment:
Perhaps the `dragHandle` icon should be declared within the `drag-handle` component?